staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/register.go

```diff
@@ -47,19 +47,6 @@ func addToGroupVersion(scheme *runtime.Scheme) error {
 	if err := scheme.AddIgnoredConversionType(&metav1.TypeMeta{}, &metav1.TypeMeta{}); err != nil {
 		return err
 	}
-	err := scheme.AddConversionFuncs(
```

Comment:
This isn't needed because below we're calling `metav1.AddToGroupVersion`, which is registering all the necessary conversions (including these ones).